### DISCUSSION TITLE
What if Harman acquired distriqt?

### DISCUSSION THREAD

**Discussion by 2jfw** [airsdk/Adobe-Runtime-Support#814] [Ideas]
Just a shower-thought if it would not be good/wise for Harman to acquire developer distriqt and have their ANE
collection be part of Harman AIR SDK, thinking to improve acceptance and features of AIR and make it more
attractive for people to use. A lot of usefull functionalities would then already be available with AIR...

Probably this will not work due to "political" reasons on both sides but I just wanted to share my thought ;)

**Comment by megajogos** [comment]
It would be amazing!!! Without distriqt, we would have given up on adobe air years ago!

**Comment by Michoko92** [comment]
I don't know if it is something Distriqt would be interested in. But at least an official partnership between Harman and Distriqt might help both to consolidate their business.

**Comment by leossmith** [comment]
Based on previous experience on Distriqt speed response to bugs and features vs Harman, I would prefer if it happened the other way around, Distriqt acquiring Harman AIR 😂
